codegenerator/cli/templates/static/codegen/src/db/Migrations.res

```diff
@@ -93,7 +93,7 @@ let deleteAllTables: unit => promise<unit> = async () => {
     BEGIN
       DROP SCHEMA IF EXISTS ${Env.Db.publicSchema} CASCADE;
       CREATE SCHEMA ${Env.Db.publicSchema};
-      GRANT ALL ON SCHEMA ${Env.Db.publicSchema} TO postgres;
+      GRANT ALL ON SCHEMA ${Env.Db.publicSchema} TO ${Env.Db.user};
       GRANT ALL ON SCHEMA ${Env.Db.publicSchema} TO public;
```

Comment:
Do we need this?

Reply:
The lines existance or the change to make it a param?

I believe we do need to grant permissions to the user that HyperIndex is using.  and we shouldn't hardcode it to Postgres if we are allowing it to be configurable by an environment variable.

Reply:
Ah, no worries. Actually you right - maybe not 🤔